framework/packages/abstract-interface/src/migrate.rs

```diff
@@ -238,24 +241,16 @@ impl<Chain: CwEnv> AbstractIbc<Chain> {
             return Ok(false);
         }
 
-        // Version change - upload both contracts
-        self.client
-            .upload_if_needed()?
-            .expect("IBC client wasm might be outdated");
-        self.host
-            .upload_if_needed()?
-            .expect("IBC host wasm might be outdated");
-
-        // Check if version is breaking
-        let version_req = semver::VersionReq::parse(&ibc_client_cw2_version).unwrap();
-        let new_version = semver::Version::parse(::ibc_client::contract::CONTRACT_VERSION).unwrap();
-        if version_req.matches(&new_version) {
```

Comment:
Why are we not using the matches here that is already implemented by the semver crate ?

Reply:
Couldn't figure out what structure of  VersionReq would work the way we want. Closest is `~` but it doesn't count pre-release as breaking versions

Reply:
Are you sure ? 
https://docs.rs/semver/latest/src/semver/eval.rs.html#3